code/game/objects/items/melee/misc.dm

```diff
@@ -482,17 +492,56 @@
 	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
 	slot_flags = ITEM_SLOT_BELT
-	force = 15
+	force = 0.001
+	reach = 3
 	w_class = WEIGHT_CLASS_NORMAL
 	attack_verb = list("flogged", "whipped", "lashed", "disciplined")
 	hitsound = 'sound/weapons/whip.ogg'
 
-/obj/item/melee/curator_whip/afterattack(target, mob/user, proximity_flag)
+/obj/item/melee/curator_whip/attack(mob/living/target, mob/living/user)
 	. = ..()
-	if(ishuman(target) && proximity_flag)
-		var/mob/living/carbon/human/H = target
-		H.drop_all_held_items()
-		H.visible_message("<span class='danger'>[user] disarms [H]!</span>", "<span class='userdanger'>[user] disarmed you!</span>")
+	var/mob/living/carbon/human/H = target
+	var/mob/living/carbon/human/U = user
+	if(ishuman(target))
+		if((user.zone_selected == BODY_ZONE_CHEST) || (user.zone_selected == BODY_ZONE_HEAD) || (user.zone_selected == BODY_ZONE_PRECISE_GROIN))
+			if(H.getarmor(type = "melee") < 16)
+				H.emote("scream")
+				H.Stun(5)
+				H.visible_message("<span class='danger'>[U] whips [H]!</span>", "<span class='userdanger'>[U] whips you! It stings!</span>")
+		if((user.zone_selected == BODY_ZONE_R_LEG) || (user.zone_selected == BODY_ZONE_L_LEG))
+			var/dist = get_dist(H, U)
+			if(dist < 2)
+				to_chat(user, "<span class='warning'>[H] is too close to trip with the whip!</span>")
+				return
+			else
+				target.Knockdown(30)
+				log_combat(user, target, "tripped", src)
+				H.visible_message("<span class='danger'>[U] trips [H]!</span>", "<span class='userdanger'>[U] whips your legs out from under you!</span>")
+			return
+		if(user.zone_selected == BODY_ZONE_L_ARM)
```

Comment:
You can probably combine the majority of this code into one here.

Reply:
due to it targeting the specific arm's item, it'd be a bit janky as i'd code it, so this is just easier to do.